config/locales/de.yml

```diff
@@ -3,7 +3,9 @@ de:
   failure_when_forbidden:
   feed_latest: RubyGems.org | Neueste Gems
   feed_subscribed: RubyGems.org | Abonnierte Gems
-  footer_about:
+  footer_about_html:
+  footer_sponsors_html: RubyGems.org is made possible by <a href="%{sponsors_list}">many sponsors</a>, with development funded by the Ruby trade association, <a href="%{rubytogether}">Ruby Together</a>.
+  footer_join_html: We need your help to fund the developer time that keeps RubyGems.org running smoothly for everyone. <a href="%{join}">Join Ruby Together today</a>.
```

Comment:
Same goes for all the Title tags that have been moved to I18N files for different languages 👍